fleet/lib/ccs-grafana/values.yaml

```diff
@@ -0,0 +1,59 @@
+---
+ingress:
+  enabled: true
+  ingressClassName: nginx
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+  hosts:
+    - cameras-grafana.${ .ClusterLabels.site }.lsst.org
```

Comment:
`cameras` feels a bit awkward to me as a name as it isn't clear if the `s` is plural or possessive.  What about `grafana.ccs.<site>.lsst.org`?

Reply:
I agree, all the names were changed, thanks!